applications/dashboard/src/scripts/pages/recoverPassword/RecoverPasswordPage.tsx

```diff
@@ -70,13 +70,13 @@ export class RecoverPasswordPage extends React.Component<IProps, IState> {
                 <div className="authenticateUserCol">
                     {pageTitle}
                     <Paragraph
-                        content={t("RecoverPasswordLabelCode", "Enter your email to continue.")}
+                        children={t("RecoverPasswordLabelCode", "Enter your email to continue.")}
                         className="authenticateUser-paragraph"
                     />
                     <form id={this.id} onSubmit={this.handleSubmit} aria-labelledby={this.pageTitleID} noValidate>
                         <Paragraph
                             className="authenticateUser-paragraph"
-                            content={getGlobalErrorMessage(this.props.requestPasswordState, ["email"])}
+                            children={getGlobalErrorMessage(this.props.requestPasswordState, ["email"])}
```

Comment:
Children is typically used the contents between the tags. If we're going to rename this property as children, we should use it that way. Otherwise, content is a better name if we're going to call it as a normal prop.

Reply:
I do like being able to pass data either through children. 
I didn't want to do too much refactoring in authenticate pages in this PR.